src/index.js

```diff
@@ -0,0 +1,9 @@
+﻿import React from "react";
+import ReactDOM from "react-dom";
+import Main from "./Main.js";
```

Comment:
Couldn't initially run because of this import. Afaict this should be
```suggestion
import Main from "./main.js";
```